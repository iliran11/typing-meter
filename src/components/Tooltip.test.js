import React from 'react';
import ToolTip from './Tooltip';
import { shallow } from 'enzyme';

const props = {
  result: 78
}
const wrapper = shallow(<ToolTip {...props} />);

describe('`ToolTip`',()=>{
  it ('renders properly',()=>{
    expect(wrapper).toMatchSnapshot();
  })
  it ()
})






