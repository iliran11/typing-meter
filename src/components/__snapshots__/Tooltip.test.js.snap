// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`ToolTip\` renders properly 1`] = `
<div
  className="tooltip"
>
  78%
</div>
`;
